dspace-uclouvain/src/main/java/org/dspace/uclouvain/packager/DSpaceUCLouvainMETSIngester.java

```diff
@@ -2,23 +2,68 @@
 
 import org.apache.logging.log4j.Logger;
 import org.dspace.authorize.AuthorizeException;
+import org.dspace.content.Bitstream;
 import org.dspace.content.DSpaceObject;
 import org.dspace.content.Item;
+import org.dspace.content.MetadataField;
+import org.dspace.content.authority.Choices;
 import org.dspace.content.crosswalk.CrosswalkException;
 import org.dspace.content.crosswalk.MetadataValidationException;
-import org.dspace.content.packager.*;
+import org.dspace.content.factory.ContentServiceFactory;
+import org.dspace.content.packager.AbstractMETSIngester;
+import org.dspace.content.packager.DSpaceMETSIngester;
+import org.dspace.content.packager.METSManifest;
+import org.dspace.content.packager.PackageParameters;
+import org.dspace.content.packager.PackageValidationException;
+import org.dspace.content.service.BitstreamService;
+import org.dspace.content.service.MetadataFieldService;
 import org.dspace.core.Constants;
 import org.dspace.core.Context;
+import org.dspace.services.ConfigurationService;
+import org.dspace.services.factory.DSpaceServicesFactory;
 import org.dspace.workflow.WorkflowException;
+import org.jdom2.Content;
+import org.jdom2.Element;
+import org.jdom2.transform.JDOMResult;
+import org.jdom2.transform.JDOMSource;
 
+import javax.xml.transform.*;
+import javax.xml.transform.stream.StreamSource;
 import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
 import java.io.IOException;
 import java.sql.SQLException;
+import java.util.*;
+import java.util.stream.Collectors;
+
+import static org.dspace.content.crosswalk.XSLTCrosswalk.DIM_NS;
 
 public class DSpaceUCLouvainMETSIngester extends DSpaceMETSIngester {
 
     private static final Logger log = org.apache.logging.log4j.LogManager.getLogger(AbstractMETSIngester.class);
+    private static final String bitstreamExtractorStylesheetConfigKey = "uclouvain.ingester.bitstreamMetadataExtractor.stylesheet";
```

Comment:
Since I don't find this key in my instance, I suppose you put it in the `local.cfg` file, then couldn't it be in the `uclouvain.cfg` file instead ? Also, what is the purpose of this file => Documentation.